contracts/IColony.sol

```diff
@@ -275,7 +275,49 @@ contract IColony {
   /// @param _role Id of the role, as defined in `ColonyStorage` `MANAGER`, `EVALUATOR` and `WORKER` constants
   /// @param _token Address of the token, `0x0` value indicates Ether
   function claimPayout(uint256 _id, uint256 _role, address _token) public;
-  
+
+  /// @notice Start next reward payout in `_token`. All funds in that are in reward pot at the time will become unavailable.
```

Comment:
"in token" implies in some Token contract, pls replace with "for" or just say "Start next `_token` reward payout"

Reply:
Also here this reads weird
>All funds in that are in reward pot at the time 
Did you mean:
"All funds in the reward pot will become unavailable"